### DISCUSSION TITLE
What are "bundled strings"?

### DISCUSSION THREAD

**Discussion by BurtHarris** [kriszyp/cbor-x#54] [Q&A]
In cbor-x's **decode.js**, we have `const BUNDLED_STRINGS_ID = 0xdff9`, and in the `read()` function, this constant is used in processing tagged values.   

I've read the [CBOR records](https://github.com/kriszyp/cbor-records) doc, but it doesn't describe the behavior of tag 0xdff9 (57337 decimal), although it does describe tags 57342-57599.   Is there some other documentation that describes this bundled strings tag?

**Comment by kriszyp** [comment]
I have not made any documentation for bundled strings, but I can try put together some documentation of how this works and is defined.

**Comment by BurtHarris** [comment]
Thanks, Kris.   It's not critical; I've been having fun reading your code.  I'm an old dog (retired coder) seeing if I can learn new tricks.  But I don't want to make work for you.  Is this something that might be discussed in working group email archives?
